scripts/aem.js

```diff
@@ -675,6 +631,39 @@ async function waitForLCP(lcpBlocks) {
   });
 }
 
+/**
+ * Loads all blocks in a section.
+ * @param {Element} section The section element
+ */
+
+async function loadSection(section, loadCallback) {
+  const status = section.dataset.sectionStatus;
+  if (status === 'initialized') {
+    section.dataset.sectionStatus = 'loading';
+    const blocks = [...section.querySelectorAll('div.block')];
+    for (let i = 0; i < blocks.length; i += 1) {
+      // eslint-disable-next-line no-await-in-loop
+      await loadBlock(blocks[i]);
+    }
```

Comment:
```suggestion
    const promises = blocks.map(block => loadBlock(block));
    await Promise.all(promises);
```
The `for` loop still makes the block loading sequential, i.e. block1, block2... blockN. If block 1 requires something remote and slow, it "blocks" all the other blocks processing. This can be avoided by parallelizing the block loading sequence.
The section is hidden anyway, no visible impact.

Reply:
It seems I have proven in the past that this does not bring any real performance benefit AND makes the block code potentially more complicated (for example, if 2 blocks uses the same resource - like metadata.json, you need a singleton to make sure it is loaded only once...). This makes this change unnecessary.